scripts/script.py

```diff
@@ -272,6 +306,8 @@ def main():
         modifyFile(modPathCountlySessionsInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyUserProfileInternal, objectOfComModification, 'mod')
         modifyFile(modPathCountlyViewsInternal, objectOfComModification, 'mod')
+        modifyFile(modPathExampleYaml, objectOfComModification, 'mod')
+        update_package("../example/lib/", 'package:countly_flutter/', 'package:countly_flutter_np/')
```

Comment:
For consistency, can we use single quotes everywhere?

Reply:
indeed